server/main.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+from fastapi import FastAPI, WebSocket
+from fastapi.responses import RedirectResponse
+from fastapi.staticfiles import StaticFiles
+
+SCRIPT = Path(__file__)
+PUBLIC_DIR = (SCRIPT / "../../public").resolve()
+
+app = FastAPI()
+
+app.mount("/static", StaticFiles(directory=PUBLIC_DIR, html=True))
+
+
+# small little redirect
```

Comment:
I'm just nitpicking, but this comment isn't really neccessary when you have the docstring